src/data/translations/id/quests.json

```diff
@@ -0,0 +1,923 @@
+    },
+    "107": {
+        "name": "Bentuk sebuah gugus tugas Kapal induk!",
+        "desc": "Miliki CV\/CVL sebagai kapal pemimpin dan 3 DD di armada utama."
+    },
+    "108": {
+        "name": "Bentuk kelas Tenryuu!",
+        "desc": "Miliki Tenryuu (\u5929\u9f8d) dan Tatsuta (\u9f8d\u7530) di armada utama."
+    },
+    "109": {
+        "name": "Bentuk kelas Sendai!",
+        "desc": "Miliki Sendai (\u5ddd\u5185), Jintsuu (\u795e\u901a) dan Naka (\u90a3\u73c2) di armada utama."
+    },
+    "110": {
+        "name": "Bentuk kelas Myoukou!",
+        "desc": "miliki Myoukou (\u5999\u9ad8), Nachi (\u90a3\u667a), Ashigara (\u8db3\u67c4) dan Haguro (\u7fbd\u9ed2) di armada utama."
+    },
+    "111": {
+        "name": "Bentuk kelas Fusou!",
+        "desc": "Miliki Fusou (\u6276\u6851) dan Yamashiro (\u5c71\u57ce) di armada utama."
+    },
+    "112": {
+        "name": "Bentuk kelas Ise!",
+        "desc": "miliki Ise (\u4f0a\u52e2) dan Hyuuga (\u65e5\u5411) di armada utama."
+    },
+    "113": {
+        "name": "Bentuk armada utama dengan BB dan CA!",
+        "desc": "Miliki sebuah BB dan 2 CA di armada utama."
+    },
+    "114": {
+        "name": "Bentuk Gugus Tugas Nagumo!",
+        "desc": "Hanya miliki Akagi (\u8d64\u57ce), Kaga (\u52a0\u8cc0), Hiryuu (\u98db\u9f8d) dan Souryuu (\u84bc\u9f8d) di armada utama."
+    },
+    "115": {
+        "name": "Bentuk Armada kedua!",
+        "desc": "Miliki kapal pemimpin di armada kedua."
+    },
+    "116": {
+        "name": "Siapkan sebuah Kapal induk pesawat laut",
+        "desc": "Miliki AV di armada utama."
+    },
+    "117": {
+        "name": "Bentuk sebuah gugus tugas kapal induk di armada kedua!",
+        "desc": "Miliki CV\/CVL dan 3 DD di armada kedua."
+    },
+    "118": {
+        "name": "Bentuk kelas Kongou!",
+        "desc": "Miliki Kongou (\u91d1\u525b), Hiei (\u6bd4\u53e1), Haruna (\u699b\u540d) dan Kirishima (\u9727\u5cf6) di armada utama."
+    },
+    "119": {
+        "name": "Bentuk armada Mikawa!",
+        "desc": "Miliki Choukai (\u9ce5\u6d77), Aoba (\u9752\u8449), Kako (\u52a0\u53e4), Furutaka (\u53e4\u9df9), Tenryuu (\u5929\u9f8d) dan kapal cepat di armada."
+    },
+    "120": {
+        "name": "Bentuk kapal perusak divisi 6!",
+        "desc": "Hanya miliki Akatsuki (\u6681), Hibiki (\u97ff), Inazuma (\u96f7) dan Ikazuchi (\u96fb) di armada utama."
+    },
+    "121": {
+        "name": "Bentuk kapal jelajah divisi 4!",
+        "desc": "Miliki Atago (\u611b\u5b95), Takao (\u9ad8\u96c4), Choukai (\u9ce5\u6d77) dan Maya (\u6469\u8036) di armada utama
+."
+    },
+    "122": {
+        "name": "Bentuk Armada Nishimura!",
+        "desc": "Miliki Fusou (\u6276\u6851), Yamashiro (\u5c71\u57ce), Mogami (\u6700\u4e0a) dan Shigure (\u6642\u96e8) di armada utama."
+    },
+    "123": {
+        "name": "Bentuk kapal Pengangkut divisi 5!",
+        "desc": "miliki Shoukaku (\u7fd4\u9db4), Zuikaku (\u745e\u9db4) dan 2 DD di armada utama."
```

Comment:
I don't understand much but wouldn't it be better if the first character is capitalized? xD